group.yml

```diff
@@ -182,3 +182,11 @@ image_build_log_scanner:
   - ppc64le.log
   - task_failed.log
   - orchestrator.log
+
+ocp_release_state:
```

Comment:
Just `release_state` sounds better? 
And would like a brief comment about how/where it is used

Reply:
Updated :+1: